scripts/fix-opera.sh

```diff
@@ -40,59 +40,59 @@ if [ -x "$(command -v opera-beta)" ]; then
   OPERA_VERSIONS+=("opera-beta")
 fi
 
-for opera in ${OPERA_VERSIONS[@]}; do
-  echo "Doing $opera"
-  EXECUTABLE=$(command -v "$opera")
+#Getting download links
```

Comment:
the diff is messed up, but I'm mainly moving the download/extract out of the loop, I'm not changing any logic